public/app/features/logs/components/LogDetails.tsx

```diff
@@ -140,10 +149,31 @@ class UnThemedLogDetails extends PureComponent<Props> {
                       wrapLogMessage={wrapLogMessage}
                       row={row}
                       app={app}
+                      disableActions={false}
                     />
                   );
                 })}
-                {!fieldsAvailable && !labelsAvailable && !linksAvailable && (
+                {fieldsWithLinksFromVariableMap?.map((field, i) => {
+                  const { keys, values, links, fieldIndex } = field;
+                  return (
+                    <LogDetailsRow
+                      key={`${keys[0]}=${values[0]}-${i}`}
+                      parsedKeys={keys}
+                      parsedValues={values}
+                      links={links}
+                      onClickShowField={onClickShowField}
+                      onClickHideField={onClickHideField}
+                      getStats={() => calculateStats(row.dataFrame.fields[fieldIndex].values.toArray())}
+                      displayedFields={displayedFields}
+                      wrapLogMessage={wrapLogMessage}
+                      row={row}
+                      app={app}
+                      disableActions={true}
```

Comment:
@svennergr The reasoning is I'm not sure actions here make sense. For example, for path here, the word path is defined in the correlation but is not in the log line at all. If you click "show this field instead of the message" the entire logline disappears for all logs - which is right, path is not a field in there.

For stats, its the same thing, it's searching through the displayed rows for a key or value but "path" is independently defined so it's not going to mean anything for any other logs.

This also gets confusing when there's multiple keys/values in the link. For show/hide, I can loop through all of them, but I'm not sure how it should work for stats.

There is probably a solution here but I think it's a separate issue to this PR and will be easier to figure out when we see what actions people want to take on their correlations. In the meantime, I think it makes sense to hide the actions.